packages/vkui/src/components/Slider/SliderThumb/SliderThumb.tsx

```diff
@@ -11,19 +23,75 @@ interface SliderThumbProps extends HasRootRef<HTMLSpanElement>, HasDataAttribute
   inputProps?: React.InputHTMLAttributes<HTMLInputElement> &
     React.RefAttributes<HTMLInputElement> &
     HasDataAttribute;
+  withTooltip?: boolean;
 }
 
 export const SliderThumb = ({
   className,
   getRootRef,
   inputProps,
+  withTooltip,
   ...restProps
 }: SliderThumbProps) => {
   const { focusVisible, onBlur, onFocus } = useFocusVisible(false);
+  const [arrowRef, setArrowRef] = React.useState<HTMLDivElement | null>(null);
+
+  const memoizedMiddlewares = React.useMemo(() => {
+    return [
+      offsetMiddleware({
+        crossAxis: 0,
+        mainAxis: 15,
+      }),
+      flipMiddleware(),
+      shiftMiddleware({ padding: 8 }),
+
+      arrowMiddleware({
+        element: arrowRef,
+        padding: 14,
+      }),
+    ];
+  }, [arrowRef]);
+
+  const {
+    x: floatingDataX,
+    y: floatingDataY,
+    placement: resolvedPlacement,
+    refs,
+    strategy: floatingPositionStrategy,
+    middlewareData: { arrow: arrowCoords },
+    update: updateTooltipPosition,
+  } = useFloating({
+    placement: 'top',
+    open: focusVisible,
+    middleware: memoizedMiddlewares,
+  });
+
+  React.useEffect(
+    function udpateTooltipPositionOnValueChange() {
+      if (focusVisible && withTooltip && inputProps && inputProps.value !== 'undefined') {
+        updateTooltipPosition();
+      }
+    },
+    [inputProps, updateTooltipPosition, focusVisible, withTooltip],
```

Comment:
```suggestion
    [inputProps.value, updateTooltipPosition, focusVisible, withTooltip],
```
можно вот так попробовать сделать

чтобы только на изменение `value` вызывался

Reply:
Вообще, прям так не получится, потому что `inputProps` может быть `undefined`. 
Но я вынес эту логику за пределы `useEffect`.
⚡ https://github.com/VKCOM/VKUI/pull/5472/commits/627bbb1cb52ff07686d09ea373c5c122daa6a118